app/uk/gov/hmrc/cbcr/package.scala

```diff
@@ -51,4 +52,10 @@ package object cbcr {
     l.map(t => t._1 -> toJsFieldJsValueWrapper(t._2))
   implicit def listJsWrapper[T: Writes](l: List[T]): List[JsValueWrapper] = l.map(toJsFieldJsValueWrapper[T])
 
+  implicit class KibanaLogger(logger: _root_.play.api.Logger) {
```

Comment:
what is this for?

Reply:
This is a part of the other PR. Good spot, should be removed.

Reply:
Removed.